applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.cpp

```diff
@@ -543,7 +543,7 @@ void UpdatedLagrangian::CalculateAndAddExternalForces(VectorType& rRightHandSide
 
 void UpdatedLagrangian::CalculateAndAddInternalForces(VectorType& rRightHandSideVector,
         GeneralVariables & rVariables,
-        double& rIntegrationWeight)
+        const double& rIntegrationWeight)
```

Comment:
Just one comment, you don't need to put & for a const double

Reply:
yeah, this one is changed because I made constant the rIntegrationWeight and thereby I need to modify the local functions too.

Reply:
@bodhinandach you should only use const if it is applicable
since you pass by ref I assume you are modifying the variable inside the function => hence it cannot be const

Reply:
@philbucher I understood. The thing is, the specific variable commented is not modified inside the function. So it’s better to make it const. Of course, I left other variables that are modified without const.